WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -44,6 +44,21 @@ def scramble_program
   def help
   end
 
+  def users_delegates_search
+    users_search(delegate_only=true)
```

Comment:
Oh, lol! That is really cute how this happens to work anyways. Fixed in f12829a884879d3a82341218df4df6a19a375bde.